geonames/geonames.php

```diff
@@ -4,18 +4,20 @@
  * Description: Use Geonames service to resolve nearest populated location for given latitude, longitude
  * Version: 1.0
  * Author: Mike Macgirvin <http://macgirvin.com/profile/mike>
- * 
- * 
+ *
+ *
  * Pre-requisite: Register a username at geonames.org
```

Comment:
Could you please add these information to the `README.md` file of the addon so that they are displayed in the admin panel.